lookout/style/typos/corrector.py

```diff
@@ -77,118 +82,88 @@ def initialize_generator(self, vocabulary_file: str, frequencies_file: str,
         Construct a new CandidatesGenerator.
 
         :param vocabulary_file: The path to the vocabulary.
-        :param frequencies_file: The path ot the frequencies.
+        :param frequencies_file: The path to the frequencies.
         :param embeddings_file: The path to the embeddings.
         :param neighbors_number: Number of neighbors of context and typo embeddings \
                                  to consider as candidates.
         :param edit_candidates: Number of the most frequent tokens among tokens on \
                                 equal edit distance from the typo to consider as candidates.
         :param max_distance: Maximum edit distance for symspell lookup.
         :param radius: Maximum edit distance from typo allowed for candidates.
-        :return: Nothing
         """
         self.generator.construct(vocabulary_file, frequencies_file, embeddings_file,
                                  neighbors_number, edit_candidates, max_distance, radius)
 
-    def train(self, typos: pandas.DataFrame, candidates: pandas.DataFrame = None,
+    def train(self, data: Union[pandas.DataFrame, str],
```

Comment:
I would prefer leaving only `DataFrame` parameters. The loading from CSV can be performed in a separate convenience function.

Reply:
This PR is all about removing those separate methods for using data from files. So should I return them back, or there must be some external wrapping?

Reply:
It is better to have separate methods to load and process the data, yes. One methods loads from csv (or whatever) to dataframe and the other works with this dataframe.